build.gradle

```diff
@@ -69,7 +69,7 @@ repositories {
 dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     compile 'com.google.guava:guava:23.0'
-    compile ('io.left.rightmesh:rightmesh-java-library:0.4.0')
+    compile ('io.left.rightmesh:rightmesh-java-library:0.5.0-JASON-SNAPSHOT') { changing = true; }
```

Comment:
Would it be worth making one-off fixed versions for things like this where we're committing it to a public repo?

I know it makes sense based on what you're working on right now just merging in an unstable dependency doesn't feel like a _great_ habit to get into. Slightly "testing-in-production"-y.

Reply:
See: https://github.com/RightMesh/Superpeer/pull/2/files#diff-c197962302397baf3a4cc36463dce5eaR127

Reply:
Yeah thats a bit tricky - because if people like Sri or David or Igor want to use the new functionality from this commit, it doesn't exist in a public version of the library yet - only exists in snapshots. Other option would be we do a 0.4.1 release with the new functionality, and then bump that way.

Reply:
Maybe we can start using tags, so we could still commit to master but the public could use tags for stable releases